docs/end_to_end_tutorials/chatbots/building_a_chatbot.md

```diff
@@ -79,143 +92,70 @@ for year in years:
     index_set[year] = cur_index
```

Comment:
Not that it's needed (since we only customized the `chunk_size`, but let's make a point to also pass in the service context when loading. It's a common mistake that others make (i.e. "I changed my LLM, but not it's not using it when I load")

```
cur_index = load_index_from_storage(storage_context, service_context=service_context)
```